test/xds/xds_client_custom_lb_test.go

```diff
@@ -124,43 +132,30 @@ func (s) TestWrrLocality(t *testing.T) {
 			// in a locality). Thus, address 1 and address 2 have 1/3 * 1/2
 			// probability, and addresses 3 4 5 have 2/3 * 1/3 probability of
 			// being routed to.
-			addressDistributionWant: []resolver.Address{
-				{Addr: backend1.Address},
-				{Addr: backend1.Address},
-				{Addr: backend1.Address},
-				{Addr: backend1.Address},
-				{Addr: backend1.Address},
-				{Addr: backend1.Address},
-				{Addr: backend2.Address},
-				{Addr: backend2.Address},
-				{Addr: backend2.Address},
-				{Addr: backend2.Address},
-				{Addr: backend2.Address},
-				{Addr: backend2.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend3.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend4.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
-				{Addr: backend5.Address},
+			addressDistributionWant: []struct {
+				addr  string
+				count int
+			}{
+				{
+					addr:  backend1.Address,
+					count: 6,
+				},
+				{
+					addr:  backend2.Address,
+					count: 6,
+				},
+				{
+					addr:  backend3.Address,
+					count: 8,
+				},
+				{
+					addr:  backend4.Address,
+					count: 8,
+				},
+				{
+					addr:  backend5.Address,
+					count: 8,
+				},
```

Comment:
Up to you, but this can be a lot easier to read if it's compressed:

```go
  {addr: backend1.Address, count:6},
  {addr: backend2.Address, count:6},
  {addr: backend3.Address, count:8},
  {addr: backend4.Address, count:8},
...
```

Reply:
Done.